tests/js/client/aql/aql-arangosearch-inverted-index-optimization.js

```diff
@@ -104,19 +85,6 @@ function testOptimizeFilterCondition() {
           RETURN [productLocationId, maxCumulative]`;
       let docs = db._query(query).toArray();
       assertEqual(docs.length, 1);
-
-      if (isServer) {
-        let plan = AQL_EXPLAIN(query);
-        plan = require("@arangodb/aql-helper").getLinearizedPlan(plan);
-
-        let hasFilter = false;
-        for (let node of plan) {
-          if (node.type === "IndexNode" && node.filter !== undefined && node.filter.name === "IN_RANGE") {
-            hasFilter = true;
-          }
-        }
-        assertTrue(hasFilter);
-      }
```

Comment:
Maybe this test still works with the client-like API?

Reply:
Pushed a change that reenables the test unconditionally.